app/styles/mixins/_layout.scss

```diff
@@ -0,0 +1,6 @@
+@mixin l-bottom {
```

Comment:
we just need to define what's what. I'm still unsure what the difference is between `--` and `__`.